src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/custom.py

```diff
@@ -855,6 +855,22 @@ def set_vm(cmd, instance, lro_operation=None, no_wait=False):
 def show_vm(cmd, resource_group_name, vm_name, show_details=False):
     return get_vm_details(cmd, resource_group_name, vm_name) if show_details \
         else get_vm(cmd, resource_group_name, vm_name)
+
+
+def update_vm(cmd, resource_group_name, vm_name, os_disk=None, no_wait=False, **kwargs):
```

Comment:
We should also open an issue to add other strongly typed convenience arguments to this command.  It's probably long overdue.